packages/typescript-estree/src/convert.ts

```diff
@@ -1563,9 +1579,10 @@ export class Converter {
           return this.createNode<TSESTree.ExportNamedDeclaration>(node, {
             type: AST_NODE_TYPES.ExportNamedDeclaration,
             source: this.convertChild(node.moduleSpecifier),
-            specifiers: node.exportClause.elements.map(el =>
-              this.convertChild(el),
-            ),
+            specifiers:
+              node.exportClause.kind === SyntaxKind.NamedExports
+                ? node.exportClause.elements.map(el => this.convertChild(el))
+                : [this.convertChild(node.exportClause.name)],
```

Comment:
this is not valid we should return different node in this case, `* as ns` and its here just as temp workaround to make it compile